test/Select.test.tsx

```diff
@@ -28,7 +30,7 @@ describe('Select', () => {
 			best: 'bar',
 		};
 
-		function SelectComponent({ initialValues, data, validator }: ComponentProps) {
+		function SelectComponent({ initialValues, data, validator, noLabel, variant }: ComponentProps) {
```

Comment:
Minor nit... I prefer to pass around positives... so instead of noLabel... prefer label

Reply:
Negative was because with a positive either existing tests need label={true} added or test snapshots will need to be regenerated. I've pushed a commit with how this would look. Tell me if you'd like for it to be reverted.